### PR TITLE
Add games

### PR DESCRIPTION
**Pull request by Siilwyn**

### CODE REVIEW

**Comment by Siilwyn** [comment]
@rowtechshio not sure about adding these games, looking at the current games they don't bring anything new to the table.

File: readme.md [hunk 1]
```diff
@@ -35,10 +35,12 @@ Pixel-level digital art, every placed pixel has importance in creating the envis
 ## Inspiration
 ### Games
 - [Super Mario World 2: Yoshi's Island](https://en.wikipedia.org/wiki/Yoshi%27s_Island) - Play as Yoshi carrying Baby Mario, platforming through a dangerous world. The painted feel and soft pastel colors create an hearth-warming handmade style that breathes the charm of a children’s book.
+- [To the Moon/Finding Paradise](http://freebirdgames.com/) - Heartfelt and tear-jerking storytelling about a mans dying wish. Beautiful soundtrack included. The pixel art is simplistic yet thorougly made, creating a warm fantasy feeling.'
```
Comment (rowtechshio) [inline readme.md]: This one is understandable, not too special or anything.

**Comment by rowtechshio** [comment]
Do I need to create a new pull request from my fork for these changes?

**Comment by Siilwyn** [comment]
I've shortened the description of Starbound and scrapped "To the Moon", hope you don't mind!

Again thank you so much, your contributions make this repository a worthy community effort!

**Pull request merged**